kolibri/plugins/management/assets/src/vue/user-page/user-create-modal.vue

```diff
@@ -57,23 +65,37 @@
       return {
         user: {
           username: '',
-          password: '',
           full_name: '',
+          password: '',
+          confirm_password: '',
         },
         role: 'learner',
       };
     },
+    computed: {
+      checkPassword() {
+        // if (this.user.password.length =< 9) {
+        //   return 'Error: too short';
+        // }
+        if (this.user.confirm_password && this.user.password !== this.user.confirm_password) {
+          return 'passwords not matched';
```

Comment:
@indirectlylit How would I go about integrating this functionality with the error message stuff in https://github.com/learningequality/kolibri/pull/372?